src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
-  margin: 0;
-}
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.header {
+  width: 100%;
+}

```

Comment:
The `.header` class is defined here but not used in the HTML file. Consider removing it if it's unnecessary or ensure it's applied to the appropriate element.